proto/repository/v1/repository.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package repository.v1;
+
+option go_package = "github.com/krok-o/krok-o/proto/repository/v1;repov1";
+
+import "google/api/annotations.proto";
+
+message Repository {
+  string name = 1;
+  int32 id = 2;
```

Comment:
I prefer the ID being the first one.
so 
int32 id = 1;
string name = 2;
etc...